libs/clients/althingi-ombudsman/src/lib/apiConfiguration.ts

```diff
@@ -25,7 +25,7 @@ export const ApiConfiguration = {
               issuer: idsClientConfig.issuer,
               clientId: idsClientConfig.clientId,
               clientSecret: idsClientConfig.clientSecret,
-              scope: config.scope,
+              scope: [],
```

Comment:
Not sure why this client library needs `autoAuth` as it is using `securityApi` to get Althingis custom token.